routes/lang/en-application.json

```diff
@@ -1,4 +1,38 @@
 {
+  "app.themesettings": "Theme settings",
+  "app.themepicker": "Theme Picker",
+  "app.themeeditor": "Theming",
+  "app.savepreset": "Save as preset",
+  "app.themingtitle": "Set the style for your course",
+  "app.themingdesc": "You can modify the appearance of your course on this page.",
+  "app.choosethemetitle": "Choose theme",
+  "app.choosethemedesc": "Select the base theme for your course and apply any saved 'preset' styles using the two dropdowns below.<br/><br/><em><b>IMPORTANT:</b> Changing the options here will replace any unsaved selections you've made in the form below; make sure there's nothing you want to keep!</em>",
```

Comment:
Please remove all `<em>`s from these strings – we don't embed the italic version of Open Sans so we get undesirable obliques.